### DISCUSSION TITLE
Question about radare2 from beginner

### DISCUSSION THREAD

**Discussion by AirShoot125** [radareorg/radare2#20826] [Q&A]
Hi to everyone, I'm a beginner in radare2 and i tried to use it in termux 
when i gave order to analyze big .so library (14.79 MB) after writing iE it doesn't shows all addresses completely (only the most recent), maybe there's any solution? Big thanks in advance to everyone who are will give an answer

**Comment by swoops** [comment]
You probably want to do `iE~..`, the `~..` is an internal "less" like command. You can also just pipe output to the real less on your system with `iE|less`.

If you want particular lines you can do `iE~search_term` or `iE~...` (3 dots) then type your search term.
